test/integration/abm_gce_defaults_on_owner_project/inspec.yml

```diff
@@ -12,23 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START anthosbaremetal_integration_abm_gce_defaults_on_owner_project_inspec_yaml]
 name: defaults_on_owner_project
 attributes:
-  - name: project_id
-    required: true
-    type: string
-  - name: bmctl_version_check
-    required: true
-    type: string
-  - name: docker_version_check
-    required: true
-    type: string
-  - name: vxlan_check
-    required: true
-    type: string
-  - name: ssh_as_tfadmin
-    required: true
-    type: string
-  - name: abm_install_check
-    required: true
-    type: string
+- name: project_id
```

Comment:
Let's make sure the formatting change doesn't impact the tests.

Reply:
I just reverted this! Id rather not mess with it, thanks for catching that! 👀